test/recurring-application-charge.test.js

```diff
@@ -78,6 +78,15 @@ describe('Shopify#recurringApplicationCharge', () => {
   });
 
   it('activates a recurring application charge', () => {
+    /**
+     * TODO: According to Shopify API the activation of a charge
+     * should return the updated charge object with the 'status'
+     * value set as 'active'. This test seems to be trying to
```

Comment:
It seems the response in docs i correct now, see https://help.shopify.com/en/api/reference/billing/recurringapplicationcharge#activate. You can take that as fixture.

Thanks.

Reply:
Roger, will do